kvnemesis: update table ID

RaduBerinde opened the pull request:
These tests hardcode a table ID of 50. This now overlaps with the
tenant_settings table. Updating to 100, which is now the first
user-created ID in a new cluster.

Release note: None

RaduBerinde commented:
Added a second commit. Feels like we are missing a constant somewhere for "ID higher than any system table in a new cluster"

ajwerner commented on pkg/kv/kvserver/reports/reporter_test.go:
```diff
@@ -487,8 +487,8 @@ func TestZoneChecker(t *testing.T) {
 	// kicks in for mapping keys to zones.
-	dbID := 50
-	t1ID := 51
+	dbID := 100
+	t1ID := 101
```
`bootstrap.TestingMinUserDescID` 
and `+1`?

ajwerner commented on pkg/kv/kvnemesis/generator.go:
```diff
@@ -250,7 +250,8 @@ func NewDefaultConfig() GeneratorConfig {
 // GeneratorDataTableID is the table ID that corresponds to GeneratorDataSpan.
-const GeneratorDataTableID = 50
+// This must be a table ID that is not used in a new cluster.
+const GeneratorDataTableID = 100
```
`bootstrap.TestingMinUserDescID` was meant to be used for this.

ajwerner commented:
I'm fine with constants for now too. It's no worse. Feel free to bors at your leisure.

RaduBerinde committed:
kvnemesis: update table ID

These tests hardcode a table ID of 50. This now overlaps with the
tenant_settings table. Updating to 100, which is now the first
user-created ID in a new cluster.

Release note: None

RaduBerinde committed:
reports: update object IDs in test

Updating object IDs in test to 100+ (50 can now overlap with a system
table).

Release note: None

RaduBerinde reviewed (commented):
TFTR!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/76459#-:-MvtGCi4-VeZs163vaw0:b-814g8k)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ajwerner)

___
*[pkg/kv/kvnemesis/generator.go, line 254 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/76459#-MvfFDu5qSPeZf-d4y5Y-r2-254:-MvtGEB5-sTShF_X3gJ0:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/9892d6b193b20ffc89b09bdc6146b4df42198ce2/pkg/kv/kvnemesis/generator.go#L254)):*
<details><summary><i>Previously, ajwerner wrote…</i></summary><blockquote>

`bootstrap.TestingMinUserDescID` was meant to be used for this. 
</blockquote></details>

Done.
___
*[pkg/kv/kvserver/reports/reporter_test.go, line 491 at r2](https://reviewable.io/reviews/cockroachdb/cockroach/76459#-MvfGMmFvuF5NZNS6fpR-r2-491:-MvtGEZV28YaTlSVOLEz:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/9892d6b193b20ffc89b09bdc6146b4df42198ce2/pkg/kv/kvserver/reports/reporter_test.go#L491)):*
<details><summary><i>Previously, ajwerner wrote…</i></summary><blockquote>

`bootstrap.TestingMinUserDescID` 
and `+1`?
</blockquote></details>

Done.


<!-- Sent from Reviewable.io -->

ajwerner reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> Reviewed 2 of 3 files at r3, 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/76459#-:-MvtKkgr04SZxcfpEsRb:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @RaduBerinde)



<!-- Sent from Reviewable.io -->

ajwerner reviewed (approved):
Needs a gazelle 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/76459#-:-MvtKv1ACqHrAKjupH1P:b-dnqshl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @RaduBerinde)



<!-- Sent from Reviewable.io -->

RaduBerinde commented:
bors r+
